Content.Server/Defusable/Systems/DefusableSystem.cs

```diff
@@ -0,0 +1,95 @@
+using Content.Server.Defusable.Components;
+using Content.Server.Explosion.EntitySystems;
+using Content.Server.Salvage;
+using Content.Shared.Defusable;
+using Content.Shared.Examine;
+
+namespace Content.Server.Defusable.Systems;
+
+/// <inheritdoc/>
+public sealed class DefusableSystem : SharedDefusableSystem
+{
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly ExplosionSystem _explosionSystem = default!;
+
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<DefusableComponent, ExaminedEvent>(OnExamine);
+    }
+
+    private void OnExamine(EntityUid uid, DefusableComponent comp, ExaminedEvent args)
+    {
+        if (!args.IsInDetailsRange)
+            return;
+
+        if (!comp.BombUsable) {
+            args.PushMarkup(Loc.GetString("defusable-examine-defused"));
+        }
+        else if (comp.BombLive)
+        {
+            args.PushMarkup(Loc.GetString("defusable-examine-live", ("time", comp.TimeUntilExplosion.ToString())));
+        }
+        else
+        {
+            args.PushMarkup(Loc.GetString("defusable-examine-inactive"));
+        }
+    }
+
+    public void TryStartCountdown(EntityUid uid, DefusableComponent comp)
+    {
+        // todo: handle countdown
+        // also might want to have admin logs
+        if (!comp.BombUsable)
+            return;
+
+        comp.BombLive = true;
+
+        Logger.Debug("it begins");
+
+        UpdateAppearance(uid, comp);
+    }
+
+    public void TryDetonateBomb(EntityUid uid, DefusableComponent comp)
```

Comment:
99% sure you can just have defusing be removing explodeontrigger component then do it like a regular explosive

Reply:
this doesnt use explodeontrigger

Reply:
yes so make it use it

Reply:
done